tests/cypress/support/step_definitions/jupyterlite/I see kernel status is.ts

```diff
@@ -0,0 +1,14 @@
+import { When } from "@badeball/cypress-cucumber-preprocessor";
+
+import MaterialDesignerPage from "../../widgets/MaterialDesignerPage";
+
+When("I see kernel status is Idle", () => {
+    const { jupyterLiteSession } = new MaterialDesignerPage().designerWidget;
+
+    cy.until({
```

Comment:
Move to browser

Reply:
Here actually we don't need to await beyond timeout for `waitForVisible`